src/coap_blockwise.h

```diff
@@ -23,6 +23,7 @@ enum golioth_status golioth_blockwise_post(struct golioth_client *client,
                                            const char *path,
                                            enum golioth_content_type content_type,
                                            read_block_cb cb,
+                                           golioth_set_cb_fn callback,
```

Comment:
Can we add a comment explaining that this function is called once with the response to the last block? I think that users could reasonably assume (without reading the implementation), that this callback is called once for _each_ block.

Reply:
Good idea. However we don't have any documentation for that module yet. So far we kind of rely on reading the implementation, since we don't provide any documentation for blockwise trasnfer module (which is private BTW). I'll leave that as an improvement for future for subsequent PRs.